front/app/components/VoteControl/index.tsx

```diff
@@ -390,13 +343,13 @@ const Downvote = styled(Vote)`
 
 interface Props {
   ideaId: string;
-  size: '1' | '2' | '3';
+  size: Size;
   unauthenticatedVoteClick?: (voteMode: 'up' | 'down') => void;
   disabledVoteClick?: (disabled_reason?: IdeaVotingDisabledReason) => void;
   setRef?: (element: HTMLDivElement) => void;
   ariaHidden?: boolean;
   className?: string;
-  style: 'border' | 'shadow' | 'compact';
+  styleType: 'border' | 'shadow';
```

Comment:
There was a weird bug going on where the style prop got converted to an object. E.g. 'border' became {0: 'b', 1: 'o', etc.). I couldn't reproduce it on master.

My best guess without further investigation is that there's name clash happening with a global variable somewhere? Still doesn't explain why it gets converted.